services/personal/api.go

```diff
@@ -62,6 +62,16 @@ func (api *PublicAPI) SetRPC(rpcClient *rpc.Client, timeout time.Duration) {
 	api.rpcTimeout = timeout
 }
 
+// Recover is an implementation of `personal_ecRecover` or `web3.personal.recover` API
```

Comment:
I think also the `web3` function is called `ecRecover` instead of just `recover` right (`web3.personal.ecRecover `)?

Reply:
good catch!